collections/misc/occurrencesearch.php

```diff
@@ -121,23 +124,23 @@ function isNumeric(inStr){
 						</div>
 					</div>
 					<div style="clear:both;padding:2px;">
-						<div style="float:left;width:130px;">Catalog #:</div>
+						<div style="float:left;width:130px;"><?php echo $LANG['CATALOG']; ?>:</div>
 						<div style="float:left;"><input name="catalognumber" type="text" value="<?= htmlspecialchars($catalogNumber, ENT_COMPAT | ENT_HTML401 | ENT_SUBSTITUTE) ?>" /></div>
 					</div>
 					<div style="clear:both;padding:2px;">
-						<div style="float:left;width:130px;">Other Catalog #:</div>
+						<div style="float:left;width:130px;"><?php echo $LANG['OTHER_CATALOG']; ?> #:</div>
```

Comment:
See comment from previous review. The # needs to be included in the lang tag. Please also add "_NO" to this language tag AND to the "CATALOG" tag.